infrastructure/ansible/roles/traffic_ops/defaults/main.yml

```diff
@@ -18,7 +18,7 @@ to_pkg_name: "{% if to_version is defined and to_version != omit %}{{ 'traffic_o
 # Postgres Superuser account to use with TO database and user creation
 postgresql_admin_user:
 postgresql_admin_user_password:
-postgresql_client_pkg_name: "{% if postgresql_version is defined and postgresql_version != omit %}{{ 'postgresql96-'+postgresql_version }}{% else %}postgresql96{% endif %}"
+postgresql_client_pkg_name: "{% if pg_major_version | int < 10 %}postgresql{{pg_major_version}}{{pg_minor_version}}-{{postgresql_version}}{% else %}postgresql{{pg_major_version}}-{{postgresql_version}}{% endif %}"
```

Comment:
If postgresql_version is undefined here, it will fail even though the major/minor are set by default.  Notice in the original the hyphen and more specific version are missing including the build are missing entirely.

Reply:
updated to address undefined postgresql_version